contrib/opencensus-ext-ocagent/opencensus/ext/ocagent/utils/__init__.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+import os
+import socket
+
+from google.protobuf.internal.well_known_types import ParseError
+from google.protobuf.timestamp_pb2 import Timestamp
+from opencensus.common.version import __version__
+from opencensus.proto.agent.common.v1 import common_pb2
+
+# Default agent endpoint
+DEFAULT_ENDPOINT = 'localhost:55678'
+
+# OCAgent exporter version
+EXPORTER_VERSION = '0.0.1'
```

Comment:
The exporter package comes with a version `https://github.com/census-instrumentation/opencensus-python/blob/master/contrib/opencensus-ext-ocagent/version.py`.
Probably consider using the same version instead of creating a new one (which is hard to maintain).

For example:
https://github.com/census-instrumentation/opencensus-python/pull/613/files#diff-cd21ef3212e9029163fd1e55142d543d

Reply:
As version.py is currently located in `contrib/opencensus-ext-ocagent/version.py` rather than contrib/opencensus-ext-ocagent/opencensus/ext/ocagent`, it makes it difficult to import it as the import path itself stats at the opencensus dir.

I imagine this is why the existing trace exporter also defines the version directly: 
https://github.com/census-instrumentation/opencensus-python/blob/3b60af770da1a90e751cb94cefcc749eb24a5768/contrib/opencensus-ext-ocagent/opencensus/ext/ocagent/trace_exporter/__init__.py#L34

I'd rather leave this out of scope of this PR and patch them together in a followup.

Reply:
Makes sense.

@c24t for awareness.